_i18n/en/install-guides/linux/index.md

```diff
@@ -35,3 +35,19 @@ To potentially fix a corrupted save:
 5. Move your original `hs_mv_prof.sav` back where it was before, replacing the one that was created by the game
 6. If this still doesn't work, delete the entire `~/.local/share/FasterThanLight/` folder
 7. ALso disable Steam Cloud Sync if you are using a version of FTL from steam, it'll frequently break the save files.
+
+#### Running FTL gives an error along the lines of "Failed to install hook"
```

Comment:
Maybe a big `#### Fedora users see below`

Reply:
Or actually maybe there should be a link to this section inside the install part saying
In the `linux/genericlinux/steam-install.md` and `linux/genericlinux/gog-install.md`
```md
[Fedora and SELinux users, an additional step is required](../../#selinux-steps)
```
_(I'm not 100% on the `../../` url part, might need to instead just include a template with this SELinux section so it shows up in both troubleshooting & genericlinux install guides)_

Then above your troubleshooting section here:
```md
<a name="selinux-steps" />
#### Running FTL gives an error along the lines of "Failed to install hook"
```
So that we have an easy to use named anchor rather than this giant one of the current header (and so it makes more sense during translation)


OR...
Maybe just have a section of this also mentioned there... but it should be called out at least somehow as part of the install guide, not just as troubleshooting.

Reply:
> Maybe a big `#### Fedora users see below` ? What do you think?

Sure, that could work well. I also think that putting it in the install guide would be good; that way, people preemptively fix it while installing. Maybe we do `#### Fedora/CentOS users see below` in the install guide, then the `Running FTL gives an error along the lines of "Failed to install hook"` in the troubleshooting section.

Reply:
Yeah lets do the callout inside the install guide, it only matters for the genericlinux gog & steam guides and not for the SteamDeck guides.